src/test/java/org/zalando/jackson/datatype/money/MonetaryAmountSerializerTest.java

```diff
@@ -269,4 +273,27 @@ public void shouldSerializeWithType() throws JsonProcessingException {
         assertThat(actual, is(expected));
     }
 
+    @Test
+    public void shouldHandleNullValueFromExpectObjectFormatInSchemaVisitor() throws JsonProcessingException {
```

Comment:
I'd prefer to have a test that makes use of mbknor-jackson-jsonschema to prove that it works.

Reply:
I didnt want to add multiple dependencies, just for one test.
But if you prefer it, I can add one

Reply:
If you put them in scope test, totally fine with me.

Reply:
added new Test in MonetaryAmountSchemaSerializerTest, which uses mbknor-jackson-jsonschema to generate the schema